test/conformance/ptp/ptp.go

```diff
@@ -279,6 +281,32 @@ var _ = Describe("[ptp]", func() {
 					waitUntilLogIsDetected(testPtpPod, 3*time.Minute, "Profile Name: test-slave")
 				})
 			})
+
+			It("Should retrieve the details of hardwares for the Ptp", func() {
+				By("Getting the version of the OCP cluster")
+
+				ocpVersion, err := getOCPVersion()
+				Expect(err).ToNot(HaveOccurred())
+				Expect(ocpVersion).ShouldNot(BeEmpty())
+
+				By("Getting the version of the PTP operator")
+
+				ptpOperatorVersion, err := getPtpOperatorVersion()
+				Expect(err).ToNot(HaveOccurred())
+				Expect(ptpOperatorVersion).ShouldNot(BeEmpty())
+
+				By("Getting the NIC details of the PTP enabled interfaces")
+
+				for _, pod := range ptpRunningPods {
+					ptpDiscoveredInterfaces := ptpDiscoveredInterfaceList(NodePtpDeviceAPIPath + pod.Spec.NodeName)
+					fmt.Println(ptpDiscoveredInterfaces)
+
+					var mapping = getNICInfo(pod)
+
+					fmt.Println(mapping)
```

Comment:
Aren't we missing this info: 
3 sh-4.4# lspci -nn | grep 85:02
85:02.0 PCI bridge [0604]: Intel Corporation Sky Lake-E PCI Express Root Port C [8086:2032] (rev 07)
use this in the report
`85:02.0 PCI bridge [0604]: Intel Corporation Sky Lake-E PCI Express Root Port C [8086:2032] (rev 07)`

Reply:
Outdated as the requirement got a bit changed.